securedrop_client/api_jobs/downloads.py

```diff
@@ -131,30 +141,5 @@ def _check_file_integrity(cls, etag: str, file_path: str) -> bool:
                 hasher.update(read_bytes)
 
         calculated_checksum = binascii.hexlify(hasher.digest()).decode('utf-8')
-        return calculated_checksum == checksum
-
-    def _decrypt_file(
-        self,
-        session: Session,
-        db_object: Union[File, Message],
-        file_path: str,
-    ) -> None:
-        file_uuid = db_object.uuid
-        server_filename = db_object.filename
-
-        # The filename contains the location where the file has been stored. On non-Qubes OSes, this
-        # will be the data directory. On Qubes OS, this will a ~/QubesIncoming directory. In case we
-        # are on Qubes, we should move the file to the data directory and name it the same as the
-        # server (e.g. spotless-tater-msg.gpg).
-        filepath_in_datadir = os.path.join(self.data_dir, server_filename)
-        shutil.move(file_path, filepath_in_datadir)
```

Comment:
where is this being done now? (the file download logic won't work on Qubes otherwise) also let's preserve the above comment since otherwise it's not obvious why this file moving is being done

Reply:
Whoops! Don't know how I missed that comment! Yes, will add back and test on qubes.